radiomanager/radiomanager/api/endpoints/handshaking.py

```diff
@@ -77,3 +80,24 @@ def _check_key(raw_key):
     def _check_name(cls, raw_name):
         if cls.ALPHANUMERIC_WITH_SPACES.fullmatch(raw_name) is None:
             raise BadRequest("Invalid radio name")
+
+class Add(Resource):
+    """
+    An example endpoint, useful for testing purposes.
+    """
+
+    def get(self):
+        key = ec.generate_private_key(ec.SECP256K1, default_backend())
+        data = key.public_key() \
+            .public_bytes(encoding=Encoding.PEM,
+                        format=PublicFormat.SubjectPublicKeyInfo)
+
+        device = Device(public_key=data,
+                            name="Space Balls",
```

Comment:
[Space Balls](https://media.giphy.com/media/LGVQJ4cQGPs8o/giphy.gif)

Reply:
Used for testing and provisioning. Leaving it in for future cases.